satpy/node.py

```diff
@@ -154,6 +162,12 @@ def __init__(self, compositor):
         """Set up the node."""
         super().__init__(compositor.id, data=(compositor, [], []))
 
+    def update_name(self, new_name):
+        """Update 'name' property and related compositor metadata."""
+        super().update_name(new_name)
+        # self.compositor.attrs.update(new_name)
+        # self.compositor.attrs['_satpy_id'] = new_name
```

Comment:
I initially thought I needed this to solve my original issue, but with the new tests it actually causes failures. I still need to add a test for my original issue so we'll see what happens.